docs/README.md

```diff
@@ -1,13 +1,16 @@
 ## Building Docs Locally
 
 ```bash
-python setup.py install (#or ./build.sh) 
+python setup.py install
 cd docs
-mkdir -p ../build/docs
```

Comment:
Minor note -- doc building on h-sim (master) is still crashing without this: 

```bash
(habitat) 
dbatra at dbatra-mbp in ~/projects/habitat-sim/docs on master
$ ./build-public.sh 
...
...
WARNING:root:classesp_1_1assets_1_1ResourceManager.xml: function parameter description doesn't match parameter names: {'shaderType': ('Indentifies the desired shader program for rendering the <a href="classesp_1_1gfx_1_1Drawable.html" class="m-doc">gfx::<wbr />Drawable</a>.', ''), 'meshID': ('Optional, the index of this mesh component stored in meshes_', ''), 'texture': ('Optional texture for the mesh.', ''), 'color': ('Optional color parameter for the shader program. Defaults to white.', '')}
Traceback (most recent call last):
  File "./m.css/documentation/python.py", line 2668, in <module>
    run(os.path.dirname(os.path.abspath(args.conf)), config, templates=os.path.abspath(args.templates))
  File "./m.css/documentation/python.py", line 2441, in run
    hooks_post_run=state.hooks_post_run)
  File "/Users/dbatra/projects/habitat-sim/docs/m.css/documentation/../plugins/m/dox.py", line 174, in register_mcss
    tagfiles=mcss_settings.get('M_DOX_TAGFILES', []))
  File "/Users/dbatra/projects/habitat-sim/docs/m.css/documentation/../plugins/m/dox.py", line 69, in init
    tree = ET.parse(os.path.join(input, tagfile))
  File "/anaconda3/envs/habitat/lib/python3.6/xml/etree/ElementTree.py", line 1196, in parse
    tree.parse(source, parser)
  File "/anaconda3/envs/habitat/lib/python3.6/xml/etree/ElementTree.py", line 586, in parse
    source = open(source, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/Users/dbatra/projects/habitat-sim/docs/../build/docs/habitat-cpp.tag'
```

Following this error up with mkdir succeeds.

Reply:
Sigh, Doxygen. Opened #630 for this.